_posts/2025-03-05-next-scala-lts-jdk.md.md

```diff
@@ -0,0 +1,30 @@
+---
+category: announcement
```

Comment:
Could you move this to the `blog` category?

We discussed these categories with Darja a few weeks ago, and we concluded that  `announcement` should only be used for release announcements. The Scala Highlights should be in `highlights` and every thing else should be in the blog category.